### ISSUE TITLE
Flank crashes on startup claiming APK on bucket doesn't exist

### ISSUE THREAD

**Issue by MatthewTPage** [Flank/flank#1098]
Flank v20.09.0 crashes on startup with my test suit, claiming that one of the apks doesn't exist:

[gs-url] from additional-app-test-apks.app doesn't exist

I redacted the gs url, but it definitely does point to a valid APK, and running the same configurations on flank v20.08.03 allows tests to be ran without this issue.

I use fladle so I can't get the raw yaml being passed to flank until flank gets far enough in its startup sequence, but here's the fladle configs I pass in (minus the specified APKs):

```
                variant.set("debug")
                serviceAccountCredentials.set(project.rootDir.resolve("firebase.json"))
                testTimeout = "40m"
                recordVideo = false
                performanceMetrics = false
                projectId = GCLOUD_PROJECT_ID
                flankVersion.set("20.09.0")
                shardTime = 120
                resultsBucket = // some bucket
                runTimeout.set("45m")
                ignoreFailedTests.set(false)
                smartFlankDisableUpload = false
                disableSharding = false
                outputStyle.set("single")

                smartFlankGcsPath = // smark flank gcs path
                devices = listOf(mapOf(
                  "model" to "Nexus7",
                  "version" to "22",
                  "locale" to "en_US"
                ))
                useOrchestrator = true
                flakyTestAttempts = 0
                legacyJunitResult.set(true)
```

**Comment by bootstraponline** [comment]
Thanks for reporting! We'll take a look.

**Comment by bootstraponline** [comment]
@MatthewTPage this is fixed in v20.09.1 (releasing today)
